kotlin/integrated-windows-authentication/README.md

```diff
@@ -1,32 +1,47 @@
-# Integrated Windows Authentication
+# Integrated windows authentication
 
-Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+Connect to an IWA secured Portal and search for maps.
 
-![Integrated windows authentication App](integrated-windows-authentication.png)
+![Image of integrated windows authentication](integrated-windows-authentication.png)
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
+
 
 ## How to use the sample
 
-1. Enter the URL to your IWA-secured portal in the edit text view at the top of the screen.
-2. Tap either the "Search Public" (which will search for portals on www.arcgis.com) or "Search Secure" (which will search your IWA-secured portal), for web maps stored on the portal.
-3. If you tap "Search Secure", you will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password.
-4. If you authenticate successfully, portal item results will display in the recycler view.
-5. Tap a web map item to display it in the map view.
+1. Enter the URL to your IWA-secured portal.
+2. Tap the button to search for web maps stored on the portal.
+3. You will be prompted for a user name, password, and domain (some platforms will use the current Windows login).
```

Comment:
Perhaps this note about the platforms should be removed or made android specific?